sysdeps/powerpc/dfpu/numdigits.h

```diff
@@ -185,7 +185,8 @@ FUNC_D (left_justify) (DEC_TYPE x)
   asm ("dxex" Q " %0,%1\n\t" : "=d"(tmp2) : "d"(rnd));
 
   d2.f = tmp2;
-  d2.l -= ADJUST;
+  if(d2.l >= ADJUST) d2.l -= ADJUST;
```

Comment:
Minor nit, formatting. Can you run this file through indent? Changes should be formatted in GNU style.

Likewise, could you add some extra tests to numdigits to verify subnormals behave? I don't think _Decimal32 will ever have issues on ppc, but it may be worth the effort for other platforms (e.g s390 or x86) which also implement this in a platform specific manner.